src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Navigation.css';
+
+function Navigation() {
+  return (
+    <React.Fragment>
+      <nav className="navigation">
+        <a className="navigation__link" href="#about">About</a>
+        <a className="navigation__link" href="#services">Services</a>
+        <a className="navigation__link" href="#contact">Contact</a>
+      </nav>
+    </React.Fragment>
```

Comment:
```suggestion
    <nav className="navigation">
      <a className="navigation__link" href="#about">About</a>
      <a className="navigation__link" href="#services">Services</a>
      <a className="navigation__link" href="#contact">Contact</a>
    </nav>
```
don't need to use extra wrapping by React fragment you already have one parent element